libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -600,20 +600,18 @@ export const parentalLeaveFormMessages: MessageDir = {
     },
     description: {
       id: 'pl.application:periods.first.period.description',
-      defaultMessage: `Þú mátt kjósa að byrja á áætluðum fæðingardegi, eða ákveðinni dagsetningu. Athugaðu að ekki er hægt að nýta réttindi til fæðingarorlofs 18 mánuðum eftir fæðingu barnsins.`,
-      description: `You can choose to start on the date of birth, or on a specific date. Please note, that your rights end 18 months after the date of birth.`,
+      defaultMessage: `Athugaðu að ekki er hægt að nýta réttindi til fæðingarorlofs 18 mánuðum eftir fæðingu barnsins.`,
+      description: `Please note, that your rights end 18 months after the date of birth.`,
```

Comment:
Did you also update the contentful's namespace at the same time? and run the extract-strings script to update the messages?

The description is not used for the English translation so it won't update it automatically on Contentful. It's also better to replace with some information's copy so the content editor has a better understanding of where/what is the message used for.

Reply:
Thanks for this info. 
I updated the descriptions and ran `yarn nx run application-templates-parental-leave:extract-strings` now. 

Do I need to do anything else for this?